### ISSUE TITLE
NVMe corruption

### ISSUE THREAD

**Issue by Lockzi** [darkhandz/XPS15-9550-Sierra#26]
Hello DarkHandz,

I used your Clover and pmset settings, yet I had another corruption.
You mention you had corruptions on El Capitan without HWPEnabler, are you sure about that you didn't have HWPEnabler?
Just for your information! 👍

Thanks for all your hard work.

**Comment by darkhandz** [comment]
Yes, I'm sure, when I using 10.11.6 without HWPEnable, it just corrupted.

**Comment by shixuev5** [comment]
@darkhandz 今天重启 突然就进不去了 然后发现系统盘崩了 还好有备份 正在还原 不知道什么原因 大家注意数据安全 在原因没有找到之前

**Comment by darkhandz** [comment]
@shixuev5 你是什么情况下崩的？睡眠唤醒后？关机重新开机的情况下？有没有做共存NVMe驱动的步骤？

**Comment by shixuev5** [comment]
@darkhand  应该算 睡眠唤醒后 唤醒后 我还用了一段时间 然后装vscode插件 各种io错误 估计系统已经崩了 我就重启 然后进不去了 坐了共存 用了快一个月 都没发生这情况 就昨天晚上。我昨天中午更新的 你的usb驱动 一下午都在用 也睡眠唤醒过 但是没问题 应该不是这问题

**Comment by shixuev5** [comment]
@darkhandz 其他都是用的你的配置 唯独 删了ssdt 用的hwp 这个应该没关系 用了快一个月 之前都没问题

**Comment by shixuev5** [comment]
@darkhandz 我已经还原了昨天晚上的系统 看看情况会不会再现

**Comment by darkhandz** [comment]
@shixuev5 好的，麻烦你了，还好你有备份
